fapi-client/src/test/scala/com/gu/facia/api/integration/IntegrationTest.scala

```diff
@@ -92,7 +92,7 @@ class IntegrationTest extends FreeSpec with Matchers with ScalaFutures with Opti
 
     "will use the provided function to adjust the query used to hydrate content" ignore {
       val adjust: AdjustSearchQuery = q => q.showTags("tone")
-      val collection = collectionResponse.right.get
+      val collection = collectionResponse.toOption.get
```

Comment:
Not essential, just wanted to note that if you update to current ScalaTest (which would involve classname/imports updates), you can use the `EitherValues` trait and do `collectionResponse.value` rather than `.toOption.get`  (thanks to https://github.com/scalatest/scalatest/pull/1895).

Reply:
Nice, this would be a good follow up improvement, but I'll leave it out since this is already a bigger PR than I'd hoped.